posts/aleo-programs-deployment.md

```diff
@@ -0,0 +1,53 @@
+# Deploying Aleo Programs in Testnet3
+
+Aleo empowers you to write and build private applications easily.
+
+As you read in the title, this guide is a walkthrough on how to deploy Aleo programs in Testnet3, this will be achieved using the Aleo SDK. We hope this blog post will help you to achieve this and discover all the potential of private applications.
+
+Before you can start deploying Aleo programs you must have the Aleo SDK installed and must have an Aleo Program created.
+
+You can follow the steps detailed in [our previous post](https://www.entropy1729.com/aleo-development-starter-pack/).
+
+## Deploying our Aleo project
+
+As said above, before deploying a program you must have a program to deploy. After creating your program, deploying it is very straightforward, just do the following:
+
+
+```bash
+cd foo
```

Comment:
Should explain that `foo` is the directory of a program that was previously created.

Reply:
yeah you should include the foo creation command earlier